storage/clickhousespanstore/mocks/spylogger_test.go

```diff
@@ -125,6 +127,37 @@ func TestSpyLogger_Error(t *testing.T) {
 	logger.AssertLogsOfLevelEqual(t, hclog.Error, logs)
 }
 
+func TestSpyLogger_Name(t *testing.T) {
+	assert.Equal(t, "spy logger", NewSpyLogger().Name())
+}
+
+func TestNotImplemented(t *testing.T) {
```

Comment:
I like the approach to test all, on the other hand testing not implementing methods in support objects (e.g. this mock) does not bring much value.